config/default.yaml

```diff
@@ -85,6 +85,77 @@ status:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: cronjobsources.sources.eventing.knative.dev
+spec:
+  group: sources.eventing.knative.dev
+  names:
+    categories:
+    - all
+    - knative
+    - eventing
+    - sources
+    kind: CronJobSource
+    plural: cronjobsources
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            data:
+              type: string
+            schedule:
```

Comment:
Thanks!

1. schedule required, if it's not present, it will be rejected;
2. true, it's hard to do schedule validation in the manifest, I put it in reconcile(), and it goes to error status if schedule is invalid.